rubocop.yml

```diff
@@ -1,6 +1,10 @@
 require:
   - rubocop-rspec
   - rubocop-performance
+  - rubocop-factory_bot
+  - rubocop-capybara
+  - rubocop-rails
+  - rubocop-rspec_rails
```

Comment:
I think these should go into `rails.yml`

Reply:
Ok will try

Reply:
```
  - rubocop-factory_bot
  - rubocop-capybara
```
Will need stay here as these are used here.